basyx.common/basyx.filerepository-backend-mongodb/src/main/java/org/eclipse/digitaltwin/basyx/core/filerepository/MongoDBFileRepository.java

```diff
@@ -68,8 +68,12 @@ public static GridFsTemplate buildDefaultGridFsTemplate(MongoTemplate mongoTempl
 
 	@Override
 	public String save(FileMetadata fileMetadata) throws FileHandlingException {
+		if (exists(fileMetadata.getFileName()))
```

Comment:
What is the purpose of this?

Reply:
If someone tries to save a file, that already exists, an FileHandlingException is thrown in the save function of the FileRepository. If this happens in the MongoDB part (in any repsoitory), the only way to solve this I found was to delete the existing file and save the new. If this is the wrong thinking and there is a better way, I am willing to change it.